core/cap-0035.md

```diff
@@ -226,6 +383,42 @@ Possible return values for the `ClawbackOp` are:
 - `CLAWBACK_UNDERFUNDED` if the `from` account does not have sufficient
   available balance of `asset` after accounting for selling liabilities.
 
+#### Create Claimable Balance Operation
+
+This proposal introduces a new failure case in the execution of the
+`CreateClaimableBalanceOp`. If the source account of the operation has a
+trustline to the asset with the `CLAWBACK_ENABLED_FLAG` flag set, the
+operation will fail with `CREATE_CLAIMABLE_BALANCE_CLAWBACK_REQUIRED` if one
+of the `claimants` `destination` is not the issuer account of the asset, or
+if the issuer account is a claimant and the `predicate` is not
+`CLAIM_PREDICATE_UNCONDITIONAL_CLAWBACK`.
+
+#### Clawback Claimable Balance Operation
+
+This proposal introduces the `ClawbackClaimableBalanceOp` operation. The
+`ClawbackClaimableBalanceOp` operation destroys a claimable balance,
+effectively returning the asset to the issuer account, burning it.
+
+The operation will only succeed if the claimable balance has in its
+`claimants` a claimant with:
+- its `destination` set to the issuer account.
+- and, its `predicate` set to include `UNCONDITIONAL_CLAWBACK`.

```

Comment:
This is kind of vague. We probably need a defined schema for including `UNCONDITIONAL_CLAWBACK`. Two options:
1. `UNCONDITIONAL_CLAWBACK | (some other predicate that doesn't include UNCONDITIONAL_CLAWBACK)`
2. Change `UNCONDITIONAL_CLAWBACK` to optionally contain another predicate that doesn't include `UNCONDITIONAL_CLAWBACK`

Reply:
I think this is good motivation for shifting to a flag on claimable balances instead of putting the issuer in the claimants: https://github.com/stellar/stellar-protocol/pull/819#discussion_r555455741.

I'll make that change to this PR.